back/api/v1/authentication.go

```diff
@@ -23,7 +23,7 @@ func setupAuthRoutes(api *Api, router *atreugo.Router) {
 }
 
 type LoginRequest struct {
-	Username string `json:"username" validate:"min=2,max=40,regexp=^[a-zA-Z]*$"`
```

Comment:
why no regexep ?

Reply:
We probably don't want to limit users on their username, just like we won't restrict them on their team names to keep compatibility with CTF Time

Frontend will have to make sure we have no XSS

Reply:
If possible check in the back and otherwise trust the front

Reply:
It's not "trust the front", rather, the front must not trust the back